### ISSUE TITLE
question about the preparing of  the data

### ISSUE THREAD

**Issue by timbryant** [avanetten/simrdwn#3]
Hey,avanetten
Thanks for you updating of the 'parse_cowc.py', and sorry for troubling you again. I have run the updated 'parse_cowc.py' again,and got the training list. But I have some questiones about the training list,which generated by the code.
Firstly, in the list of my data,since I put the xxx.png and the xxx_Annotated_Cars.png and the xxx_Annotated.xcf together,so the list contains all the files. Does the list  need the xxx_Annotated_Cars.png and the xxx_Annotated.xcf in the following data transformed steps?
Sencodly, in the code 'preprocess_tfrecords.py', I found the code , 'label_file_full = image_file_full.split('.')[0].replace('images', 'labels') + '.txt'', we have generated the training list in the 'parse_cowc.py', but haven't generated the label.txt. I think the label.txt is necessary, but how can we generate the label.txt ? and what the use of .csv,which is generated by the  'parse_cowc.py'?
I am very interested in your work, and looking forward to your reply
Thank you for your great work again

**Comment by timbryant** [comment]
Hi,jarhed
Thanks for your reply.
you said your training seems to be going along now, have you created .tfrecord? I also generate the label.txt, which is generated by my own added code,and created .tfrecord,but when I train the network, I don't know what the  command is, here is my command as following, I use the .tfrecord.
 "python ./core/simrdwn.py /
 --framework yolt /
--mode train /
--outname inception_v2_car /
--yolt_object_labels_str car /
--tf_cfg_train_file ./yolt/cfg/ave_dense.cfg /
--train_tf_record ./data/trainTest1030_nes/car/car_train.tfrecord/
 --train_val_tf_record ./data/trainTest1030_nes/car/car_val.tfrecord/
 --weight_dir ./yolt/input_weights /
--weight_file yolov3.weights/
 --yolt_train_images_list_file ./data/trainTest1030_nes/car/cowc_train_list.txt /
--label_map_path ./data/class_labels_car.pbtxt /
--nbands 3 /
--max_batches 30000 /
--batch_size 64/
 --subdivisions 16 /
--gpu 2"
,but it shows the error :
nohup: redirecting stderr to stdout  
./yolt/darknet: 1: ./yolt/darknet: Syntax error: "(" unexpected
I wander how you train the network and use the .csv,which generated by the parse_cowc.py , and have you ever seen this problem?
Thank you so much!

**Comment by jarhed** [comment]
You need to compile darknet.

Go to ./yolt/ inside your docker image and run make to build the binary

**Comment by timbryant** [comment]
Hi,jarhed
What is your training going on ? I have trained  the network, but the network didn't work very well. I know where the question is, I use the whole image as the input of the network, so the network didn't work, I read the paper again, the paper said use the appropriate cutout of the large image as the input of the network. Do you have some ideas about how to generate the appropriate cutout of the large image?
Thanks a lot

**Comment by slash-fury** [comment]
@timbryant if you look at core/slice_im.py, you can see how an image can be sliced into multiple appropriately-sized images. The annoying part, for training, is that you'll also have to generate new label files with x, y, w, h all scaled to the sliced images.

**Comment by avanetten** [comment]
We've been busy updating the code and writing papers, and updated examples will be uploaded in the near future.

**Comment by avanetten** [comment]
Take a look at https://github.com/avanetten/simrdwn/blob/master/core/prep_data_cowc.py, hopefully this will provide everything you need prepare training data
